cluster/vm-template-windows2012r2-nopvc.yaml

```diff
@@ -0,0 +1,69 @@
+apiVersion: v1
+kind: Template
+metadata:
+  name: windows2012r2-vm-template
+  annotations:
+    description: "OCP KubeVirt Microsoft Windows Server 2012 R2 VM template"
+    tags: "kubevirt,ocp,template,windows"
+  labels:
+    kubevirt.io/os: win2k12r2
+    miq.github.io/kubevirt-is-vm-template: "true"
+objects:
+- apiVersion: kubevirt.io/v1alpha1
+  kind: OfflineVirtualMachine
+  metadata:
+    name: ${NAME}
+    labels:
+      kubevirt-ovm: ovm-${NAME}
```

Comment:
It's probably better to use the `kubevirt.io/` prefix just for consistency sake.